dashboard/app/controllers/registrations_controller.rb

```diff
@@ -91,7 +91,8 @@ def update_params(params)
       :user_type,
       :school,
       :full_address,
-      :terms_of_service_version
+      :terms_of_service_version,
+      races: []
```

Comment:
Apologies, I missed this before I merged - my feeling is that since this is causing a real user-facing issue, fixing it is the top priority. I'm not opposed to whitelisting other fields, but I'm not too familiar with the other serialized attributes and where they're used, so I'd probably put them in another PR if need be:

serialized_attrs %w(ops_first_name ops_last_name district_id ops_school ops_gender races)